gradle.properties

```diff
@@ -1,6 +1,6 @@
 # configuring groupVal rather than group prevents a circular dep in mavenPublishing
 groupVal=io.consensys.protocols.shomei
-version=2.3.3-SNAPSHOT
+version=2.4.0
```

Comment:
Might need to make this `develop` or conditionl on publishing docker + artifactory?